CMakeLists.txt

```diff
@@ -62,10 +62,8 @@ endif()
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake/recipes/")
 
-# check if Perl is installed (required by HDF5 but not set as required there)
-find_package (Perl REQUIRED)
```

Comment:
This was explicitly added there to make sure the compilation breaks if Perl is not available. I think this should stay. Or is there a replacement for that?

Reply:
where is the error? if perl is required for hdf5, we should add it there

Reply:
We can also add it to the hdf5 repo. Currently, if Perl was not found some header files will not be generated and the generation will break at compile time, not during CMake configuration. This already caused confusion several times.

Reply:
this should fix it https://github.com/polyfem/paraviewo/commit/b41de37e807d010f72a592b609c0614d9cabd3cd

Reply:
this should fix the problem
https://github.com/polyfem/paraviewo/commit/ef005d4686e425b5e3ff8bf839aa3fbea5fd5a4f